ttt.c

```diff
@@ -3,9 +3,15 @@
 #include <stdlib.h>
 #include <string.h>
 
+#define board_size 4
```

Comment:
Capitalize the macro identifiers. That is, use `BOARD_SIZE` rather than `board_size`.